int32 timestamp
int32 lastAction
int8[] lastX
int8[] X
float32 speed
float32 load
float32 encoder
Prediction[] lastPredictions